File: linguagem_dart/encapsulamento/academia/academia.dart
class Academia {
  // Atributos ca classe Academia
  String _nome = "";
  String _cnpj = "";

  // Getter e setter de nome
  String get getNome => _nome;

  set setNome(String nome) {
    _nome = nome;
  }

  // Getter e setter de cnpj
  String get getCnpj => _cnpj;

  set setCnpj(String cnpj) {
    _cnpj = cnpj;
  }

  // Construtor da classe
  Academia(String nomeDaAcademia, String cnpjDaAcademia) {
    setNome = nomeDaAcademia;
    setCnpj = cnpjDaAcademia;
  }
}

File: linguagem_java/encapsulamento/academia/Academia.java
package encapsulamento.academia;

public class Academia {
    // Atributos da classe Academia
    private String nome;
    private String cnpj;

    // Getter e setter de nome
    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    // Getter e setter de cnpj
    public String getCnpj() {
        return cnpj;
    }

    public void setCnpj(String cnpj) {
        this.cnpj = cnpj;
    }

    // Construtor da classe Academia
    Academia(String nomeDaAcademia, String cnpjDaAcademia) {
        setNome(nomeDaAcademia);
        setCnpj(cnpjDaAcademia);
    }
}
